GOVERNANCE.md

```diff
@@ -68,10 +83,10 @@ The Buildpack Authors' Tooling team is responsible for helping buildpack authors
 
 * [libcnb](https://github.com/buildpacks/libcnb)
 
-## Emeritus Team
+### Emeritus Team
 Serving as a member of an open source project requires a huge amount of work that cannot be sustained indefinitely. The emeritus team includes members of the project to whom we will always be grateful, but who no longer actively participate in the project.
 
-Project members may graduate to the emeritus team by a supermajority vote from the core team.
+Project members may graduate to the emeritus team by a supermajority vote from Team Leads.
```

Comment:
I am assuming this doesn't include TOC members or other team leads?

Reply:
It shouldn't: Team Leads shouldn't be able to vote TOC members out

Reply:
```suggestion
Project members may graduate to the emeritus team by a supermajority vote from the TOC.
```

Reply:
Playing a scenario in my head:

If a maintainer is being removed (let's say by absence), based on this wording:

> maintainers can be removed by a supermajority of Team Leads.

... Team Leads would vote for such removal.

_Then_, based on the latest proposed changes:

> Project members may graduate to the emeritus team by a supermajority vote from the TOC.

... separately we may ask to have the TOC vote to add the no-longer maintainer to the Emeritus team.

Did I get that right? Is that what we expect and want the process to look like?

Reply:
I think ideally I imagine we would want to have something like -

- Contributors <- Supermajority of maintainers of that team
- Maintainers <- Supermajority of team leads
- Team leads/TOC <- Supermajority of TOC

Reply:
resolved by https://github.com/buildpacks/community/pull/165/commits/198808695694e0c6be96c63608bfc2813845d68d

went with TOC+Leads for maintainers (appointment and removal), but happy to change later to team leads if this starts to feel onerous